Raymond-Osy/iReporter
# chore(back-end): LFA feedback

#### What does this PR do?
- It is for the feedback given by the LFA for my backend 

#### Description of Task to be completed?
- Use a proper context as a response for unknown routes
- Follow the rules stated for API specification
- 'createdOn' should be an integer
- Remove image from the request body as it is an optional feature
- Follow naming convention for route.js and test.js
- Add more validations for routes
- Remove the PUT request
- Rename redflagid to id
- Add status in the body of the request
- Create a middleware for validations
- Fix code coverage issue
- Write more tests to ensure code coverage

#### How should this be manually tested?
- Check the new branch to ensure that all feedback listed in the descriptions above are reflected

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#162439563](https://www.pivotaltracker.com/n/projects/2226946)

#### Screenshots (if appropriate)
N/A

#### Questions:
- Please I need a feedback with regards my Travis and coveralls if it is now properly set up according to recommended standards
